### DISCUSSION TITLE
[FEATURE]: Transactions included in withdrawal, transfer id

### DISCUSSION THREAD

**Discussion by twegrewicz** [etsy/open-api#710] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

The accountant needs information about exactly what operations are included in the transfer (for which orders, what part of a given order is included in the transfer). This is extremely important for companies with full accounting.

### Describe the solution you'd like

The most important is the payout ID visible in the transfer title, and information about the purchase transactions included in it.
ideally, you could assign your internal number to your order.

**Comment by etsyachristensen** [comment]
We are closing the issues board in favor of the Discussions system. We'll convert this to a new discussion in our Ideas category.

**Comment by vlladin** [comment]
Will this feature be considered at all? The lack of mapping between transactions and disbursements is causing a lot of headaches for the European sellers with a lot of sales, mainly because of how accounting has to be done at the end of the month. 

I managed to find a manual and painful way of guessing which transactions are included in a disbursement, but it’s not always working. I read on the forum that a lot of people are struggling with this. 

Any help would be greatly appreciated.

**Reply by etsyachristensen** [reply]
I don't believe it has been discussed yet internally. I have no information whether this will be done or not.
